src/sentry/auth/partnership_config.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class ChannelName(Enum):
+    FLY_IO = "fly.io"
+
+
+SPONSOR_OAUTH_NAME = {ChannelName.FLY_IO: "Fly IO"}
```

Comment:
They may want this string representation as "Fly.io" rather than "Fly IO" if this appears in any publicly visible area.